juju/juju
# Fix application facades

Restore facades on application 16 that were incorrectly dropped and increment facade to 17

## Checklist


## QA steps

Verify unit tests pass

Verify DestroyUnits facade can be called using application facade 15 or 16
